imports/lib/models/base.ts

```diff
@@ -35,8 +36,22 @@ class Base<T extends BaseType> extends Mongo.Collection<T> {
     this.name = name;
     this.tableName = tableName;
 
-    // All models have standard roles
-    this.attachRoles(`mongo.${name}`);
+    // Let admins get away with absolutely anything from the client.
+    // This is an affordance intended for use by a human trying to fix things
+    // in production with the `Models` global manually populated by the
+    // `loadFacades()` global function, and these allow rules should never be
+    // relied upon by any client-side code that is part of the application.
```

Comment:
Hmm. This makes me wonder if we should figure out some way to make these rules off by default? We could have a Meteor method that enables them per-session or something like that and upgrade `loadFacades()` to, like, `enableAdminMode()` or whatever. Not a today problem but it makes me wonder if we can systematically prevent accidentally relying on these in the future.

Reply:
That's a nice idea -- we could have these rules also check a global boolean or something that we can toggle via a Meteor method `breakGlass()`, and maybe have it automatically set a 10-minute timer to flip the boolean back to false?

Or it could just be a feature flag we can toggle from `/setup`